metarParser-commons/src/test/java/io/github/mivek/internationalization/ArchitectureTest.java

```diff
@@ -0,0 +1,23 @@
+package io.github.mivek.internationalization;
+
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.junit.ArchUnitRunner;
+import com.tngtech.archunit.lang.ArchRule;
+import org.junit.runner.RunWith;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+/**
+ * @author mivek
+ */
+@RunWith(ArchUnitRunner.class)
+@AnalyzeClasses(packages = "io.github.mivek.internationalization", importOptions = { ImportOption.DoNotIncludeTests.class })
+public class ArchitectureTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ArchitectureTest' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/mivek/MetarParser/pullRequest?prid=5569897)